components/worktemplates/components/menu.tsx

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import classNames from 'classnames';
+import {useIntl} from 'react-intl';
+import {useDispatch, useSelector} from 'react-redux';
+import styled from 'styled-components';
+
+import {injectDevData} from 'mattermost-redux/actions/worktemplates';
+import {Category, WorkTemplate} from '@mattermost/types/worktemplates';
+import {GlobalState} from '@mattermost/types/store';
+
+import {worktemplates as workTemplateDevData} from '../dev_data';
+
+import UseCaseMenuItem from './menu/use_case';
+
+const Categories = styled.div`
+    h2 {
+        margin: 0;
+        padding: 8px 16px;
+        font-weight: 600;
+        font-size: 12px;
+        line-height: 16px;
+        letter-spacing: 0.02em;
+        text-transform: uppercase;
+    }
+
+    ul {
+        list-style: none;
+        padding: 0;
+        width: 176px;
+    }
+`;
+
+const CategoryButton = styled.button`
+    width: 155px;
+    padding: 10px 16px;
+    border: 0;
+    background: none;
+    text-align: left;
+
+    &.selected {
+        background: rgba(var(--denim-button-bg-rgb), 0.04);
```

Comment:
There's some spots here that I think need to match the user's current theme, not the denim theme since I think these appear in the regular app and not in an unthemed page (like the System Console)

Reply:
the variable comes from `_css_variables.scss` and its value has no other equivalent, it also appears to be themed (bg color of the modal on page 2 and 3):

![image](https://user-images.githubusercontent.com/785518/205374105-c41e375d-e6c5-4861-9269-0a72267058b2.png)